scripts/spec/kickstart_generator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+require 'kickstart_generator'
+
+describe Build::KickstartGenerator do
+  describe "#run" do
+    def build_base
+      data_file_path("kickstart_generator")
```

Comment:
You probably want to use [mktmpdir](http://ruby-doc.org/stdlib-2.0.0/libdoc/tmpdir/rdoc/Dir.html) instead.